requirements_test_all.txt

```diff
@@ -49,9 +49,6 @@ aiounifi==4
 # homeassistant.components.notify.apns
 apns2==0.3.0
 
-# homeassistant.components.zha
-bellows==0.7.0
```

Comment:
Since I don't see additions to this file, I think the tests won't pass?

Reply:
This was addressed